### PR TITLE
Add test-like output when test file does not exist

### PR DESCRIPTION
**Pull request by sotojuan**
This helps with #575. Kinda.

First as @novemberborn said:

> If we can agree that these are not fatal errors then I think we can fix up the reporters.

But if we agree to do them in the reporters, I fixed up the extraneous new lines and added a cross for `AvaError` type of exceptions.

Here's how test on a nonexistent file and a regular one looks like:

screen shot 2016-04-03 at 8 08 08 pm

Also there are a lot of places I could've chosen to remove newlines from, so tell me if there are any better ones (I did this quickly to see what you guys think!).

### CODE REVIEW

File: lib/reporters/mini.js [hunk 1]
```diff
@@ -131,7 +132,7 @@ MiniReporter.prototype.reportCounts = function () {
 		status += '\n   ' + colors.todo(this.todoCount, 'todo');
 	}
 
-	return status.length ? status : '\n';
+	return status.length ? status : '';
```
Comment (novemberborn) [inline lib/reporters/mini.js]: Could just return `status` now.

File: lib/reporters/mini.js [hunk 2]
```diff
@@ -200,7 +201,7 @@ MiniReporter.prototype.finish = function () {
 
 MiniReporter.prototype.write = function (str) {
 	cliCursor.hide();
-	this.currentStatus = str + '\n';
+	this.currentStatus = String(str);
```
Comment (novemberborn) [inline lib/reporters/mini.js]: Why this change?

Comment (sotojuan) [inline lib/reporters/mini.js]: `xo` rule.

Comment (novemberborn) [inline lib/reporters/mini.js]: Well before it was concatenating with a newline. That's been removed so presumably you could just assign `this.currentStatus = str`?

**Comment by sotojuan** [comment]
Thanks for the tips, fixed.

**Comment by sindresorhus** [comment]
Thanks @sotojuan :)
